source/math/FloatBox2D.ooc

```diff
@@ -50,16 +56,16 @@ FloatBox2D: cover {
 	intersection: func (other: This) -> This {
 		left := this left > other left ? this left : other left
 		top := this top > other top ? this top : other top
-		width := ((this right < other right ? this right : other right) - left) maximum(0.0f)
-		height := ((this bottom < other bottom ? this bottom : other bottom) - top) maximum(0.0f)
+		width := Float maximum(0, (this right < other right ? this right : other right) - left)
+		height := Float maximum(0, (this bottom < other bottom ? this bottom : other bottom) - top)
```

Comment:
```ooc
this bottom < other bottom ? this bottom : other bottom
```
This is exactly what `maximum` does:
```ooc
    // FloatExtension.ooc
	maximum: static func(first: This, second: This) -> This {
		first > second ? first : second
	}
```
So it would be better to write
```ooc
Float maximum(0, (Float maximum(this right, other right) - left))
```

This goes for the rest of the file as well. It's probably necessary to add a suffix `~two` to `FloatExtension`'s `maximum` and `minimum` functions so that it looks just like in `IntExtension`, and the reason I didn't use these `maximum` and `minimum` functions to begin with is probably that they didn't work and I didn't know why, but that's no problem anymore, so let's use those functions and make this code a little neater.